qemu/tests/win_virtio_driver_install_from_update.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import time
+
+from virttest import error_context
+from virttest import utils_misc
+from virttest.utils_windows import wmic
+
+
+QUERY_TIMEOUT = 360
+INSTALL_TIMEOUT = 360
+OPERATION_TIMEOUT = 120
+
+
+def _pnpdrv_info(session, name_pattern, props=None):
+    cmd = wmic.make_query("path win32_pnpsigneddriver",
+                          "DeviceName like '%s'" % name_pattern,
+                          props=props, get_swch=wmic.FMT_TYPE_LIST)
+    return wmic.parse_list(session.cmd(cmd, timeout=QUERY_TIMEOUT))
```

Comment:
This seems totally same with https://github.com/autotest/tp-qemu/pull/2482/files#diff-b6094f287a31dd775ff73923834113dbe30a4b3beb38e93ab3e848035d74fb1bR25, should be reused.

Reply:
yes,  and I add win_driver_utils.py to provider first on this patch to let this it go ahead, thanks